### PR TITLE
Approval permission for 1.32 Release

### PR DESCRIPTION
**Pull request by chanieljdan**
As the 1.32 Release Docs lead, I would like to add myself to the approver list for the dev-1.32 branch

### CODE REVIEW

**Comment by sftim** [comment]
> As the 1.32 Release Docs lead, I would like to add myself to the approver list for the dev-1.21 branch

Guessing you meant dev-1.32 there!

**Review by sftim — COMMENTED** [review]
This PR seems to target the wrong branch.

/approve cancel
I assume that you meant to send this PR in against _dev-1.32_

**Comment by tengqm** [comment]
No. I don't think this should target dev-1.32. The change to OWNER_ALIAS is temporary, but the whole site wide.
/approve
again.

**Comment by tengqm** [comment]
xref: #47390
xref: #44709

**Comment by chanieljdan** [comment]
> xref: #47390 xref: #44709

Yeah these were the examples I followed, as well as [#26114](https://github.com/kubernetes/website/pull/26114). Let me know if it is indeed the wrong branch though, I can update it!

**Comment by chanieljdan** [comment]
Hey @sftim I followed the examples above and targeted the PR to the main branch, let me know if it is indeed the wrong branch, but if not, could I get your lgtm?

**Comment by sftim** [comment]
> could I get your lgtm

I still think this is the wrong approach. I don't object to another person adding LGTM if other people are confident I'm wrong.

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -66,6 +66,7 @@ aliases:
     - tengqm
     - Princesso # RT 1.31 Docs Lead
     - drewhagen # RT 1.31 temp acting Docs lead
```
[the remark this answers is not in the document]
Comment (chanieljdan) [reply OWNERS_ALIASES]: Done!

**Pull request merged**